SC_redesign/Crop_and_Soil/soil/phosphorus_cycling/fertilizer.py

```diff
@@ -0,0 +1,240 @@
+from typing import Optional
+from math import log, exp
+
+from SC_redesign.Crop_and_Soil.soil.soil_data import SoilData
+
+"""
+This module incorporates equations from the SurPhos model to simulate the Phosphorus from fertilizer applied to the soil
```

Comment:
Small nitpick: we're not simulating phosphorus, but some process related to phosphorus.

```suggestion
This module incorporates equations from the SurPhos model to simulate the [cycling of?] Phosphorus from fertilizer applied to the soil
```

Reply:
I have updated the top-level docstring, is much more descriptive